server/app/services/ti/TrustedIntermediaryService.java

```diff
@@ -0,0 +1,97 @@
+package services.ti;
+
+import com.google.common.base.Preconditions;
+import com.google.common.base.Strings;
+import com.google.common.collect.ImmutableList;
+import forms.UpdateApplicantDob;
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+import java.util.Locale;
+import java.util.Optional;
+import javax.inject.Inject;
+import models.Account;
+import models.Applicant;
+import models.TrustedIntermediaryGroup;
+import play.data.Form;
+import repository.SearchParameters;
+import repository.UserRepository;
+import services.DateConverter;
+import services.applicant.exception.ApplicantNotFoundException;
+
+public class TrustedIntermediaryService {
```

Comment:
Please mark this class as final to make the decision to subclass explicit rather than implicit.

Reply:
@swatkat1 this comment was resolved without being addressed